eslint.config.js

```diff
@@ -0,0 +1,316 @@
+import jqueryConfig from "eslint-config-jquery";
+import importPlugin from "eslint-plugin-import";
+import globals from "globals";
+
+export default [
+	{
+
+		// Only global ignores will bypass the parser
+		// and avoid JS parsing errors
+		// See https://github.com/eslint/eslint/discussions/17412
+		ignores: [
+			"test/data/json_obj.js"
+		]
+	},
+
+	{
+		files: [
+			"eslint.config.js",
+			"Gruntfile.cjs",
+			"test/node_smoke_tests/commonjs/**",
+			"test/node_smoke_tests/module/**",
```

Comment:
These also shouldn't be grouped, I think.

Reply:
Again, I think we're good because of the package.json files.